students/MichaelM/lesson2/Lesson2_Ex2_FizzBuzz.py

```diff
@@ -0,0 +1,30 @@
+# cd C:\Users\v-micmcd.Redmond\Dropbox\UW_Python\git\Self_Paced-Online\students\MichaelM\lesson2
+# cd C:\Users\geekw\Dropbox\UW_Python\git\Self_Paced-Online\students\MichaelM\lesson2
+# run Lesson2_Ex2_FizzBuzz.py
+# git status
+# git add Lesson2_Ex2_FizzBuzz.py
+# git commit
+# git push
+# goto https://github.com/geekwriter2/Self_Paced-Online/tree/master/students
+# click Pull request > new pull request
+# import name_main
+# run Lesson2_Ex1.py
+# from Natasha: python uses snake_case convention for var names (and functions)> horizontal_line
+
+def buzzles_and_fizzles(top_pot):
+        for i in range(top_pot):
+            if (i%3) == 0:
```

Comment:
if i == 15, then this line will evaluate to True and will print "Fizz"; however the requirement is to print FizzBuzz if it divisible by both 3 and 5.

You need to move up your check on line 20 to be the first thing you check for.